packages/core/src/components/menu/menu.css.ts

```diff
@@ -0,0 +1,123 @@
+import { createVar, style } from '@vanilla-extract/css';
+import type { RecipeVariants } from '@vanilla-extract/recipes';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { foregrounds } from '~/styles/mixins/foreground.css';
+import { interaction } from '~/styles/mixins/interactions.css';
+import { typography } from '~/styles/mixins/typography.css';
+import { layerStyle } from '~/styles/utils';
+import { vars } from '~/styles/vars.css';
+
+const contentWidth = createVar({ inherits: false, syntax: '*' }, 'menu-content-width');
+const contentHeight = createVar({ inherits: false, syntax: '*' }, 'menu-content-height');
+
+export const content = layerStyle('components', {
+    display: 'flex',
+
+    flexDirection: 'column',
+    border: `0.0625rem solid ${vars.color.border.normal}`,
+
+    borderRadius: vars.size.borderRadius['300'],
+    boxShadow: vars.shadow.md,
+
+    backgroundColor: vars.color.background.normal,
+    padding: vars.size.space['050'],
+    width: '12.5rem',
+    minWidth: contentWidth,
+
+    overflowY: 'auto',
+    vars: {
+        [contentWidth]: 'var(--radix-dropdown-menu-trigger-width)',
+    },
+});
+
+export const item = recipe({
+    base: [
+        interaction({ type: 'roving' }),
+        typography({ style: 'body2' }),
+        foregrounds({ color: 'normal' }),
+        layerStyle('components', {
+            position: 'relative',
+
+            display: 'flex',
+            alignItems: 'center',
+            alignSelf: 'stretch',
+            justifyContent: 'space-between',
+            gap: vars.size.space['100'],
+
+            border: 'none',
+
+            borderRadius: vars.size.borderRadius['300'],
+            cursor: 'pointer',
+            paddingRight: vars.size.space['075'],
+            paddingLeft: vars.size.space['250'],
+            paddingBlock: vars.size.space['050'],
+
+            height: vars.size.dimension['400'],
+        }),
+    ],
+
+    variants: {
+        disabled: {
+            true: layerStyle('components', { opacity: 0.32, pointerEvents: 'none' }),
+        },
+    },
+});
+
+export const separator = layerStyle('components', {
+    flexShrink: 0,
+    marginBlock: vars.size.space['050'],
+    backgroundColor: vars.color.border.normal,
+    height: '0.0625rem',
+});
+
+export const subContents = layerStyle('components', {
+    display: 'flex',
+    flexDirection: 'column',
+
+    border: `0.0625rem solid ${vars.color.border.normal}`,
+    borderRadius: vars.size.borderRadius['300'],
+    boxShadow: vars.shadow.md,
+
+    backgroundColor: vars.color.background.normal,
+    padding: vars.size.space['050'],
+
+    minWidth: contentWidth,
+    maxHeight: contentHeight,
+    overflowY: 'auto',
+
+    vars: {
+        [contentWidth]: 'var(--radix-dropdown-menu-trigger-width)',
+        [contentHeight]: 'var(--radix-dropdown-menu-content-available-height)',
```

Comment:
```suggestion
 // top 
 const RADIX_TRIGGER_WIDTH = 'var(--radix-dropdown-menu-trigger-width)';
 const RADIX_CONTENT_HEIGHT = 'var(--radix-dropdown-menu-content-available-height)';
  
// ...

        [contentWidth]: 'var(--radix-dropdown-menu-trigger-width)',
        [contentHeight]: 'var(--radix-dropdown-menu-content-available-height)',
```

These values appear to be duplicates. How about creating them as constants variables and managing them?
In addition, I suggest using SNAKE_CASE, which is our team's convention for constant variables.

Reply:
Thank you for the feedback. After thinking it over, I realized those variables were unnecessary and have removed them. I appreciate your input!